import-export-cli/integration/devFirst_test.go

```diff
@@ -357,3 +357,55 @@ func TestUpdateDocAndImageOfAPIOfExistingAPI(t *testing.T) {
 	//Validate that document has been updated
 	testutils.ValidateAPIWithIconIsExported(t, args, testutils.DevFirstDefaultAPIName, testutils.DevFirstDefaultAPIVersion)
 }
+
+// Test sequence update as a super tenant user with Internal/devops role
```

Comment:
Shall we add the same test case for tenant users with Internal/devops role as we do in the other test cases?